### PR TITLE
LPS-179688 As an Instance Administrator, I want to be able to configure that if the users have the option to give their password at registration

### PR DESCRIPTION
**Pull request by thabet91**

### CODE REVIEW

File: modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_user_registration/create_account_user.jsp [hunk 1]
```diff
@@ -156,6 +156,7 @@ portletDisplay.setURLBack(backURL);
 			</div>
 
 			<div class="form-group">
```
Comment (hhtank) [inline modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_user_registration/create_account_user.jsp]: should we hide the `div` form-group as well?

File: modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_user_registration/create_account_user.jsp [hunk 2]
```diff
@@ -156,6 +156,7 @@ portletDisplay.setURLBack(backURL);
...
+			<c:if test="<%= PrefsPropsUtil.getBoolean(company.getCompanyId(), PropsKeys.LOGIN_CREATE_ACCOUNT_ALLOW_CUSTOM_PASSWORD, PropsValues.LOGIN_CREATE_ACCOUNT_ALLOW_CUSTOM_PASSWORD) %>">
```
Comment (hhtank) [inline modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_user_registration/create_account_user.jsp:159]: Indentation is not aligned, i suppose we should run `gw formatSource` though. Should we?

File: modules/apps/login/login-web/src/main/resources/META-INF/resources/create_account.jsp [hunk 3]
```diff
@@ -224,27 +224,28 @@ renderResponse.setTitle(LanguageUtil.get(request, "create-account"));
 			</div>
 
 			<div class="form-group">
```
Comment (hhtank) [inline modules/apps/login/login-web/src/main/resources/META-INF/resources/create_account.jsp]: May hide the form-group and also formatSource

**Review by hhtank — CHANGES_REQUESTED** [review]

**Comment by thabet91** [comment]
@hhtank  please check the changes

**Comment by hhtank** [comment]
@thabet91 Can you help check check again, i still do see indentation not good so far

image

**Comment by thabet91** [comment]
@hhtank I did run formatSource and I retried. It looks formated on my local IDE ( check attached picture). I ran it again and there was no change to the file. This could be an issue about how git render the code view.
for

**Comment by hhtank** [comment]
@thabet91 The changes look good so far. I'm fine with this PR. I'll close this and tell you how to send PR to AppSec team.

**Pull request closed without merging**